packages/loot-core/src/server/budget/goaltemplates.js

```diff
@@ -64,6 +64,11 @@ async function processTemplate(month, force) {
           num_applied++;
           await setBudget({ category: category.id, month, amount: to_budget });
         }
+        if (applyErrors.length > -1) {
```

Comment:
When would this be false?

(Also instead of the loop you can do `errors = errors.concat(applyErrors)` just like the code above)

Reply:
Done.